src/lint.js

```diff
@@ -74,7 +79,8 @@ function lintStr(specsStr) {
       const next = linked.seriesNext ?
         linkedList.find(p => p.shortname === linked.seriesNext) :
         null;
-      const isLast = !next || next.seriesComposition === "delta";
+      const isLast = !next || next.seriesComposition === "delta" ||
```

Comment:
probably not a problem today, but with the `fork` model, there isn't really a sense to "last" anymore, since there can be different parallel forks.

Reply:
That's true and that creates a problem. Today, we have `seriesPrevious` and `seriesNext` properties that create a linear set of specs in a series. If there is more than one fork of a spec, I don't know what `seriesNext` of the last spec that is not a fork in a series should point to. Nothing? Should there be a `forks` property to easily tell that a spec has forks and what shortname they have?

Reply:
I think a short term solution would be not to mark forks as `seriesNext` (so in our case, wasm-js-api would always be the latest) and indeed document forks in a new property.